application_sandbox/simple_compute/add_numbers.comp

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#version 430
+#include "inputs.h"
+
+layout (local_size_x = 128, local_size_y = 1, local_size_z = 1) in;
```

Comment:
Should probably make 128 a constant you get from inputs.h as well.  It needs to be common between shader and host. At the very least as a preprocessor definition.